API.md

```diff
@@ -265,6 +280,13 @@ lastChangeTime: Date
 ```
 
 
+## `TrialUser`
+```
+email: String
+plainPassword: String
```

Comment:
```suggestion
password: String
```
I feel like `plain` is superfluous here; it'd be `passwordHash` otherwise

Reply:
`plainPassword` is more consistent with the rest of the (symfony) application: For the login to work, I have to implement an interface containing a `getPassword()` method which should return the hashed password.

Reply:
But I don't really mind

Reply:
I'd prefer `password`, even if it's a little awkward that `getPassword()` wouldn't return that on your side, as long as it's not much of an issue :)

Reply:
will do